subprojects/internal-integ-testing/src/main/groovy/com/google/common/truth/DiffUtilsAccess.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.common.truth;
+
+import java.util.List;
+
+/**
+ * This static util class exists to allow us to use default access level methods on Google Truth's {@code DiffUtils} class;
```

Comment:
Is there an upstream issue to expose this as API? If we ever go modular in tests this is going to break, though I doubt that's going to happen soon.

Reply:
Actually, after re-reading the PR body, 100% think that you should just add the class instead, with attribution.

Reply:
This would be fine with me.  Sterling, Kyle, what do you think?

Reply:
I'm going to make this change and just copy the class and remove the dep.